### PR TITLE
add scss framework and navbar

### PR DESCRIPTION
**Pull request by dkhgh**
screen shot 2015-06-05 at 17 53 03

### CODE REVIEW

**Comment by dkhgh** [comment]
ready

File: Gemfile [hunk 1]
```diff
@@ -18,6 +18,9 @@ gem "gravtastic"
 
 gem "cancancan", "~> 1.10"
 
+gem 'bourbon'
+gem 'neat'
+gem 'refills'
```
Comment (fatboypunk) [inline Gemfile]: :dog:

File: app/views/refills/_navigation.html.erb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        <%= link_to "Sign up with Facebook", user_omniauth_authorize_path("facebook"), class: "sign-up" %>
+      <% end %>
+
+      <% if user_signed_in? %>
+      <ul id="js-navigation-menu" class="navigation-menu show">
+        <li class="nav-link more"><a href="javascript:void(0)"><%= current_user.email %></a>
+          <ul class="submenu">
+            <li><%= link_to "Profile", current_user %></li>
+            <li>
+            <%= link_to('Logout', destroy_user_session_path, :method => :delete) %>
```
Comment (fatboypunk) [inline app/views/refills/_navigation.html.erb]: I18n :dog:

**Comment by fatboypunk** [comment]
:see_no_evil:

**Pull request merged**